lib/chat_app/contexts/messages.ex

```diff
@@ -5,9 +5,11 @@ defmodule ChatApp.Contexts.Messages do
   def get_message(id), do: Repo.get(Message, id)
 
   def update_message(%Message{} = message, attrs) do
-    message
-    |> Message.changeset(attrs)
-    |> Repo.update()
+    updated_message =
+      message
+      |> Message.changeset(attrs)
+      |> Repo.update()
+    updated_message
```

Comment:
Why don't we return `message` directly? Does `Repo.update()` not return the object properly? I did the same thing in my `get_room_*` functions because I couldn't properly get the keys I needed, but I'm not sure it's needed here